native-schema-registry/csharp/AWSGsrSerDe/AWSGsrSerDe/common/GlueSchemaRegistryConstants.cs

```diff
@@ -0,0 +1,16 @@
+namespace AWSGsrSerDe.common
+{
+    public class GlueSchemaRegistryConstants
+    {
+        public const string AvroRecordType = "avroRecordType";
+        public const string CacheItemExpirationTime = "cacheItemExpirationTime";
+
+        public enum DataFormat
```

Comment:
TODO: We need to explore if this can exposed from underlying Java library to avoid duplication across languages.

Reply:
Another option we have is generate this enum from resource files and use that as a source of truth. We might be able to achieve the same goal with less effort. I have added a todo for that